src/chocolatey/infrastructure.app/configuration/ChocolateyConfiguration.cs

```diff
@@ -384,6 +384,8 @@ public sealed class SourcesCommandConfiguration
         public string Username { get; set; }
         public string Password { get; set; }
         public int Priority { get; set; }
+        public string Certificate { get; set; }
+        public string CertificatePassword { get; set; }

```

Comment:
I see the two authentication methods as complimentary so it's quite possible for the passwords to be different.